apps/cli/scripts/verify-s04.sh

```
@@ -0,0 +1,247 @@
+else
+  fail "3 — prepublishOnly hook missing from package.json"
+fi
+
+echo ""
+echo "--- npm pack dry-run ---"
+
+# ----------------------------------------------------------------
+# Check 4 — npm pack --dry-run lists expected files
+# ----------------------------------------------------------------
+dry_out=$(npm pack --dry-run 2>&1)
+file_count=$(echo "$dry_out" | grep -c "npm notice" || true)
```

Shellcheck (0.11.0)

[warning] file_count appears unused. Verify use (or export if used externally).

(SC2034)